src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -609,6 +606,7 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             location,
             onAnnotationSelect,
         } = this.props;
+        const annotationFileVersionId = getProp(file_version, 'id');
```

Comment:
Can `null` get pushed as part of the history path? Should we do an identity check in the if on line 618?